CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## 2.30.0-rc.0 / 2021-09-08
+
+* [FEATURE] **experimental** TSDB: Snapshot in-memory chunks on shutdown for faster restarts. Behind `--enable-feature=memory-snapshot-on-shutdown` flag. #7229
+* [FEATURE] **experimental** Scrape: Configure scrape interval and scrape timeout via relabeling using `__scrape_interval__` and `__scrape_timeout__` labels respectively. #8911
+* [FEATURE] Scrape: Add `scrape_timeout_seconds` and `scrape_sample_limit` metric. Behind `--enable-feature=extra-scrape-metrics` flag to avoid additional cardinality by default. #9247 #9295
+* [FEATURE] Scrape: Add `--scrape.timestamp-tolerance` flag to adjust scrape timestamp tolerance when enabled via `--scrape.adjust-timestamps`. #9283
+* [ENHANCEMENT] Remote Write: Improve per shard buffering. #8921
```

Comment:
maybe `Remote Write: reduce memory used by data buffering within shards.`? @csmarchbanks

Reply:
I don't think it will actually change memory all that much. `Remote Write: Improve throughput when sending exemplars` would be more accurate for what will change to a user I think?

Reply:
Doing `Remote Write: Improve throughput when sending exemplars`